packages/core/src/mpi/get-patient-by-demo.ts

```diff
@@ -49,9 +53,13 @@ export const getPatientByDemo = async ({
     [matchingPersonalIdentifiersRule, matchingContactDetailsRule],
     foundPatients.map(patientToPatientMPI),
     normalizedPatientDemo,
-    SIMILARITY_THRESHOLD
+    SIMILARITY_THRESHOLD,
+    false
   );
 
+  if (matchingPatients.length > 1) {
+    log("WARNING: matchPatients in getPatientByDemo returning more than one matched patient!");
```

Comment:
Let's use `capture.message(..., { level: "warning" })`, please. We don't monitor logs. Can also log _in addition to_ capturing a msg.

[Example](https://github.com/metriport/metriport/blob/214d938c7cbbbbc3026685ca1a3d3df7e76ad75b/packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts#L42).

Reply:
Was looking for this! Log didn't feel right either (i.e. essentially yelling into the abyss) :).

Reply:
Done.